src/server/models/career.model.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+let careerSchema = new Schema({
+    schoolName: String,
```

Comment:
I'm not sure why school is in position? Do you mean company? All the fields in this object can probably be the same as the one in linkedin api:

https://developer.linkedin.com/docs/fields/positions